src/global.css

```diff
@@ -1,10 +1,30 @@
-:root {
-  --color-primary: 0deg 0% 0%;
-  --color-secondary: 0deg 0% 24%;
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
You should not be doing this I believe. This results in duplicate classes. Could you check this issue? I remember the duplication issue. The way we add Tailwind in Astro is a bit different.

Reply:
Ok I will check this

Reply:
@suyash5641 in fact check this in the docs: https://docs.astro.build/en/guides/integrations-guide/tailwind/#applybasestyles

Reply:
@yaralahruthik  So what if we remove @tailwind base; @tailwind components; @tailwind utilities; and instead use this 

:root {
  --color-primary: 0deg 0% 0%;
  --color-secondary: 0deg 0% 24%;
  --color-hover: 0deg 0% 97%;
  --foreground: 0deg 0% 0%;
  --background: 0deg 0% 100%;
}

.dark {
  --color-hover: 0deg 0% 15%;
  --color-secondary: 255deg 100% 100%;
  --foreground: 0deg 0% 100%;
  --background: 0deg 0% 0%;
}

html {
  scrollbar-gutter: stable;
}

body {
  background-color: hsl(var(--background));
  color: hsl(var(--foreground));
}

Reply:
I am not sure but I think this could cause issues. I am also not sure how Tailwind directives actually work. Have to do some reading.

> To disable this default behavior, set applyBaseStyles to false. This can be useful if you need to define your own base.css file (to include a [@layer directive](https://tailwindcss.com/docs/functions-and-directives#layer), for example). This can also be useful if you do not want base.css to be imported on every page of your project.

But just disabling that default injection and providing our own global.css seems the better idea and more aligned with how others user Tailwind.

Reply:
I will also search again and lets see if we can find a proper solution for this .